scripts/restore

```diff
@@ -69,12 +75,20 @@ ynh_script_progression --message="Reinstalling dependencies..." --weight=1
 # Define and install dependencies
 ynh_install_app_dependencies $pkg_dependencies
 
-#================================================
+#=================================================
+# FIND AND OPEN A PORT
+#=================================================
+ynh_script_progression --message="Open 445 port..." --weight=1
+
+ynh_exec_warn_less yunohost firewall allow Both 445 --no-upnp
+
+
+#=================================================
 # INTEGRATE SERVICE IN YUNOHOST
-#================================================
+#=================================================
 ynh_script_progression --message="Integrating service in YunoHost..." --weight=1
 
-yunohost service add smbd --description="Samba service" --log="/var/log/smbd/smbd.log"
+yunohost service add smbd --description="Samba service" --log="/var/log/smbd/smbd.log" --needs_exposed_ports 445
```

Comment:
This params will be used by remote diagnosis to remotely test if the port is open and the diagnosis will say it should be open. BUT we don't want to open that port on the internet.

Reply:
```suggestion
# Don't add --needs_exposed_ports 445 , we don't want a remote diagnosis here
yunohost service add smbd --description="Samba service" --log="/var/log/smbd/smbd.log"
```